api/controllers/service_api/dataset/segment.py

```diff
@@ -36,6 +36,10 @@ def post(self, tenant_id, dataset_id, document_id):
         document = DocumentService.get_document(dataset.id, document_id)
         if not document:
             raise NotFound("Document not found.")
+        if document.indexing_status != "completed":
+            raise NotFound("Document is already completed.")
```

Comment:
`already completed`? Shouldnt it be `Document is still under processing` or `Document is not completed yet` ?